proto/eth/v2/beacon_block.proto

```diff
@@ -85,6 +113,69 @@ message BeaconBlockAltair {
   BeaconBlockBodyAltair body = 5;
 }
 
+message BeaconBlockBodyMerge {
+  // The validators RANDAO reveal 96 byte value.
+  bytes randao_reveal = 1 [(ethereum.eth.ext.ssz_size) = "96"];
+
+  // A reference to the Ethereum 1.x chain.
+  v1.Eth1Data eth1_data = 2;
+
+  // 32 byte field of arbitrary data. This field may contain any data and
+  // is not used for anything other than a fun message.
+  bytes graffiti = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+
+  // Block operations
+  // Refer to spec constants at https://github.com/ethereum/consensus-specs/blob/dev/specs/core/0_beacon-chain.md#max-operations-per-block
+
+  // At most MAX_PROPOSER_SLASHINGS.
+  repeated v1.ProposerSlashing proposer_slashings = 4 [(ethereum.eth.ext.ssz_max) = "16"];
+
+  // At most MAX_ATTESTER_SLASHINGS.
+  repeated v1.AttesterSlashing attester_slashings = 5 [(ethereum.eth.ext.ssz_max) = "2"];
+
+  // At most MAX_ATTESTATIONS.
+  repeated v1.Attestation attestations = 6 [(ethereum.eth.ext.ssz_max) = "128"];
+
+  // At most MAX_DEPOSITS.
+  repeated v1.Deposit deposits = 7 [(ethereum.eth.ext.ssz_max) = "16"];
+
+  // At most MAX_VOLUNTARY_EXITS.
+  repeated v1.SignedVoluntaryExit voluntary_exits = 8 [(ethereum.eth.ext.ssz_max) = "16"];
+
+  // Sync aggregate object to track sync committee votes for light client support. 
+  v1.SyncAggregate sync_aggregate = 9;
+
+  // Execution payload: the embedded execution payload of the block [New in Merge]
+  v2.ExecutionPayload execution_payload = 10;
+}
+
+
+// The execution payload in a `BeaconBlockBody`.
+message ExecutionPayload {
+    // Execution block header fields
+    bytes parent_hash = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+    bytes coinbase = 2 [(ethereum.eth.ext.ssz_size) = "20"]; //'beneficiary' in the yellow paper
+    bytes state_root = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+    bytes receipt_root = 4 [(ethereum.eth.ext.ssz_size) = "32"]; //'receipts root' in the yellow paper
+    bytes logs_bloom = 5 [(ethereum.eth.ext.ssz_size) = "256"];
+    bytes random = 6 [(ethereum.eth.ext.ssz_size) = "32"]; //'difficulty' in the yellow paper
+    uint64 block_number = 7; //'number' in the yellow paper
+    uint64 gas_limit = 8;
+    uint64 gas_used = 9;
+    uint64 timestamp = 10;
+    bytes extra_data = 11 [(ethereum.eth.ext.ssz_max) = "32"];
+    bytes base_fee_per_gas = 12 [(ethereum.eth.ext.ssz_size) = "32"]; //base fee introduced in EIP-1559, little-endian serialized
+    bytes block_hash = 13 [(ethereum.eth.ext.ssz_size) = "32"]; // Hash of execution block
+    //repeated bytes transactions = 14  [(ethereum.eth.ext.ssz_size) = "?,?", (ethereum.eth.ext.ssz_max)  = "16384,1048576"];
+    repeated Transaction transactions = 14 [(ethereum.eth.ext.ssz_max) = "16384"];
```

Comment:
This is probably outpdated

Reply:
I just rebased to test on testnet, if we are going to merge this on kintsugi then I'll update these fields.